src/selectors/index.js

```diff
@@ -54,16 +55,17 @@ export {
   getOpenTopBarMenu,
   getProject,
   getRequestedFocusedLine,
+  isClassroomExportInProgress,
   isCurrentlyValidating,
   isCurrentProjectSyntacticallyValid,
   isEditingInstructions,
   isExperimental,
+  isGapiReady,
   isGistExportInProgress,
   isRepoExportInProgress,
-  isClassroomExportInProgress,
   isSnapshotInProgress,
   isTextSizeLarge,
-  isUserAuthenticated,
   isUserTyping,
   makeGetProjectPreview,
+  makeUserIsAuthenticatedWith,
```

Comment:
As a corollary with my other comment on this, we would _not_ export this from the `selectors` module—we’d just use it to create concrete selectors that are then exported.